Source/Projections/Projections.csproj

```
@@ -19,6 +19,7 @@
         <ProjectReference Include="../Events/Events.csproj" />
         <ProjectReference Include="../Events.Processing/Events.Processing.csproj" />
         <ProjectReference Include="../Protobuf/Protobuf.csproj" />
+        <ProjectReference Include="..\Resources\Resources.csproj" />
```

Just to keep style consistent.
```suggestion
        <ProjectReference Include="../Resources/Resources.csproj" />
```